pages/osx/dockutil.md

```diff
@@ -0,0 +1,32 @@
+# dockutil
+
+> Manage macOS dock items.
+> More information: <https://github.com/kcrawford/dockutil>.
+
+- Add an application to the end of the current user's dock:
+
+`dockutil --add {{/path/to/application}}`
```

Comment:
```suggestion
`dockutil --add {{path/to/application}}`
```
Does this have to be an absolute path? And should it point to the executable itself? Or some other type of file?

Reply:
It does not need to be absolute, just an application file